public/static/locales/fr/setup-security-questions.json

```diff
@@ -18,5 +18,8 @@
     },
     "required": "Required field[FR]",
     "answerLength": "Doit être au moins 4 caractères."
+  },
+  "success": {
+    "title": "[FR] Your security questions have been saved"
```

Comment:
@anikbrazeau added a toast message success here --- will need a translation + feel free to modify the content.